### DISCUSSION TITLE
Person height in Preferences->People using Imperial does not reflect input

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1254] [General]
Person height using the Imperial measurement (not tested for Metric) is overstated in the People List by 1 or 2 inches over the Person dialog box.
![Image1](https://github.com/mytourbook/mytourbook/assets/106929478/e0c9fd26-7ff6-490d-98b5-5a011981b6f5)

**Comment by wolfgang-ch** [comment]
Fixed in https://github.com/mytourbook/mytourbook/issues/1255
